src/db.js

```diff
@@ -32,7 +32,9 @@ class Database {
     }
 
     async query(query, bindings) {
-        return (await this.conn.raw(query, bindings))[0];
+        const isSPCall = query.match(/^call\s/ig);
+        const result = await this.conn.raw(query, bindings);
+        return (isSPCall === null ? result[0] : result[0][0]);
```

Comment:
Double-check this logic:
```suggestion
        const isSPCall = null !== query.match(/^call\s/ig);
        const result = await this.conn.raw(query, bindings);
        return (isSPCall ? result[0][0] : result[0]);
```